Production/nginx.tmpl

```diff
@@ -309,6 +308,14 @@ server {
 	location /lnd-rest/btc/ {
 		rewrite ^/lnd-rest/btc/(.*) /$1 break;
 		proxy_pass http://lnd_bitcoin:8080/;
+	}
+		{{ end }}
+		{{ if (eq $serviceName "lnd_bitcoin_rtl") }}
+	location /rtl/ {
+		proxy_pass http://lnd_bitcoin_rtl:3000/;
+	}
+	location /api/ {
+		proxy_pass http://lnd_bitcoin_rtl:3000/api/;
```

Comment:
this both me a bit. `/api/` is a kind of path we want to use in BTCPay in the future.

Can't their front end use a relative path to the api, so that if we go to `/rtl/` then it will query `/rtl/api` ?